docs/migration/seidb_archive_migration.md

```diff
@@ -0,0 +1,177 @@
+# SeiDB Archive Migration Guide
+
+## Overview
+SeiDB is the next generation of chain storage in SeiV2.
+One issue for running SeiDB on archive nodes is that we need to keep the full state of the chain, so we can't
+state sync it and clear out previous iavl data.
+
+In order to run an archive node with SeiDB, we need to run a migration from iavl to sei-db.
+
+The overall process will work as follows:
```

Comment:
We probably need to add a requirement section, since we need extra disk storage for the migration data

Reply:
Added